### PR TITLE
 Add libbz2-dev to buildDeps

### LINKED ISSUE #39: Should slim also install libbz2-dev
When trying to `pip install pytz` i get the following error message:

    tarfile.CompressionError: bz2 module is not available

Imo libbz2-dev should be installed and included before building python.

### PR DESCRIPTION
**Pull request by denibertovic**
Closes #39

@tianon @yosifkit  I'm not sure what to do about the tests ... that's mentioned in the original issue. Is this good to go as is? (and then refactor the tests as needed?)

### CODE REVIEW

File: 3.4/slim/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ ENV PYTHON_VERSION 3.4.3
 RUN gpg --keyserver pool.sks-keyservers.net --recv-keys 97FC712E4C024BBEA48A61ED3A5CA953F73C700D
 
 RUN set -x \
-	&& buildDeps='curl gcc libc6-dev libsqlite3-dev libssl-dev make xz-utils zlib1g-dev' \
+	&& buildDeps='curl gcc libc6-dev libsqlite3-dev libssl-dev make xz-utils zlib1g-devi libbz2-dev' \
```
Comment (tianon) [inline 3.4/slim/Dockerfile]: `devi`? :+1:

(you must be a Vim user! :smile:)

Also, worth noting that these are sorted alphabetically. :wink:

**Comment by tianon** [comment]
You've tested to make sure this actually _fixes_ #39, right, and aren't just taking our word that it's probably the right fix? :smile:

**Comment by tianon** [comment]
(and yeah, adding a test to verify this in the future too to the official images tests would be a good idea, but orthogonal)

**Comment by denibertovic** [comment]
@tianon yeah vim user. Sorry about that typo :) I'll address your comments later today.

Oh, and yes I've verified it actually fixes the issue because I've bumped into it myself, and after adding this it worked.

**Comment by denibertovic** [comment]
I should mention that I used the 2.7 version.

**Comment by tianon** [comment]
Cool, sounds good :+1:  Just give a shout when you've got the fixes in. :smile:

**Comment by denibertovic** [comment]
@tianon @yosifkit updated! I will add tests in a separate PR for https://github.com/docker-library/official-images/issues/560

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
LGTM

**Pull request merged**